src/components/stake/StakePoolCommissions.tsx

```diff
@@ -58,10 +59,18 @@ const PoolCommissions: FC = () => {
                     : lastValue
             );
 
-            setList((lastValue) => [
-                ...lastValue,
-                ...data.stakingPoolFeeHistories,
-            ]);
+            setList((lastValue) =>
```

Comment:
I noticed an issue with the fee histories when you disconnected and then reconnected your wallet. In this case the fee histories were being duplicated. This fixes the issue.